Foundation/NSLocale.swift

```diff
@@ -177,18 +177,18 @@ public let NSLocaleScriptCode: String = "kCFLocaleScriptCodeKey"
 public let NSLocaleVariantCode: String = "kCFLocaleVariantCodeKey"
 public let NSLocaleExemplarCharacterSet: String = "kCFLocaleExemplarCharacterSetKey"
 public let NSLocaleCalendar: String = "kCFLocaleCalendarKey"
-public let NSLocaleCollationIdentifier: String = "collation"
+public let NSLocaleCollationIdentifier: String = "kCFLocaleCollationIdentifierKey"
 public let NSLocaleUsesMetricSystem: String = "kCFLocaleUsesMetricSystemKey"
 public let NSLocaleMeasurementSystem: String = "kCFLocaleMeasurementSystemKey"
 public let NSLocaleDecimalSeparator: String = "kCFLocaleDecimalSeparatorKey"
 public let NSLocaleGroupingSeparator: String = "kCFLocaleGroupingSeparatorKey"
 public let NSLocaleCurrencySymbol: String = "kCFLocaleCurrencySymbolKey"
-public let NSLocaleCurrencyCode: String = "currency"
-public let NSLocaleCollatorIdentifier: String = "" // NSUnimplemented // NSString
-public let NSLocaleQuotationBeginDelimiterKey: String = "" // NSUnimplemented // NSString
-public let NSLocaleQuotationEndDelimiterKey: String = "" // NSUnimplemented // NSString
-public let NSLocaleAlternateQuotationBeginDelimiterKey: String = "" // NSUnimplemented // NSString
-public let NSLocaleAlternateQuotationEndDelimiterKey: String = "" // NSUnimplemented // NSString
+public let NSLocaleCurrencyCode: String = "kCFLocaleCurrencyCodeKey"
```

Comment:
Tests are almost always a good thing! I'd say that verifying the constants are quite reasonable to validate since we have a nearly concrete obligation to keep them the same (for archiving and other fringe cases) for the darwin version of Foundation.